src/index.html

```diff
@@ -4,9 +4,387 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home" class="logo">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo__nav"
+          >
+        </a>
+      </div>
+      <nav class="nav">
```

Comment:
```suggestion
      </div>
      
      <nav class="nav">
```
add blanks between multiline blocks, it will increase readability